proposals/nnnn-add-ability-to-reuse-a-syncfilename-for-a-putfile.md

```diff
@@ -0,0 +1,54 @@
+# Add Ability to Reuse a SyncFileName for a PutFile
+
+* Proposal: [SDL-NNNN](nnnn-add-ability-to-reuse-a-syncfilename-for-a-putfile.md)
+* Author: [Nicole Yarroch](https://github.com/NicoleYarroch)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+Currently, when a developer tries to reuse the same `syncFileName` for a `PutFile` with image data, the HMI is not updated to show the new image. This is due to Core only sending a notification to the HMI when the `PutFile` is a system file. To remedy this issue, Core needs to send a notification to the HMI when it receives any type of `PutFile`.    
+
+## Motivation
+
+Reusing a `syncFileName` for a `PutFile` is useful for developers because they do not have to come up with unique names for dynamically updated images, and it saves them the extra step of having to deleting the old data as the new data overwrites the old.
```

Comment:
Update to:
Reusing a `syncFileName` for a `PutFile` is useful for developers because they do not have to come up with unique names for dynamically updated images, and it saves them the extra step of having to **delete** the old data as the new **data will overwrite** the old.